starters/nuxt3-pinia-vuetify/test/utils.ts

```diff
@@ -0,0 +1,5 @@
+/**
+ * Choosing to export testing library through a utility file instead to avoid major refactorings and allowing custom configuration.
+ * Please see: https://testing-library.com/docs/react-testing-library/setup as reference
+ */
+export * from '@testing-library/vue'
```

Comment:
Do we want this? As we are using Vitest I think we could delete this file and remove the library

Reply:
I am currently using it for referencing items in the DOM and also triggering dom events

Reply:
You still want to use @testing-library with vitest, they solve separate problems